storm-core/test/clj/org/apache/storm/metrics_test.clj

```diff
@@ -238,143 +238,4 @@
       )))
 
 
-(deftest test-builtin-metrics-1
```

Comment:
Do we have a replacement for these tests?

Reply:
No.  This seemed like an integration test.  I'm not sure how valuable this is.